packages/extension/src/background/index.ts

```diff
@@ -107,7 +118,15 @@ browser.browserAction.onClicked.addListener(() => {
   browser.tabs.create({ url, active: true });
 });
 
-browser.runtime.onInstalled.addListener(async () => {
+browser.runtime.onInstalled.addListener(async (details) => {
+  if (details.reason === 'update') {
+    updateCompanionAlerts({ displayCompanionPopup: true });
+  }
```

Comment:
What happens when a new users installs -> then updates to a next version?
Perhaps we should lock in to only this version or something specific?

Let me know what you think

Reply:
Good point. I think we need to add another check if the value is explicitly false, meaning the user has seen the popup (as it was undefined initially), we won't turn this to true. WDYT?